virtcontainers/kata_agent.go

```diff
@@ -265,6 +266,11 @@ func KataAgentKernelParams(config KataAgentConfig) []Param {
 		params = append(params, Param{Key: "agent.trace", Value: config.TraceType})
 	}
 
+	if config.ContainerPipeSize > 0 {
+		containerPipeSize := strconv.FormatUint(uint64(config.ContainerPipeSize), 10)
+		params = append(params, Param{Key: "agent.container_pipe_size", Value: containerPipeSize})
```

Comment:
Can you use a `const` exported from the annotations package to avoid hard-coding this string?

Reply:
Done!